lib/elixir/lib/protocol.ex

```diff
@@ -715,7 +715,14 @@ defmodule Protocol do
     quote bind_quoted: [built_in: __built_in__()] do
       any_impl_for =
         if @fallback_to_any do
-          quote do: unquote(__MODULE__.Any).__impl__(:target)
+          quote do
+            try do
+              unquote(__MODULE__.Any).__impl__(:target)
+            rescue
+              UndefinedFunctionError ->
+                nil
+            end
```

Comment:
This is a change in behaviour. If you fallback to any, then it is always required, we never potentially return nil.

Reply:
I see. Also note that we would still be raising in `impl_for/1`  if `@fallback_to_any true` and no implementation for Any exists